app/assets/javascripts/chessboard.js

```diff
@@ -0,0 +1,80 @@
+$(function() {
+    // draw the chess board
+    for (var y = 0; y <= 7; y++) {
+        $("#chess-board").append(
+          "<tr>" +
+            "<td data-x='0' data-y='" + y + "'></td>" +
+            "<td data-x='1' data-y='" + y + "'></td>" +
+            "<td data-x='2' data-y='" + y + "'></td>" +
+            "<td data-x='3' data-y='" + y + "'></td>" +
+            "<td data-x='4' data-y='" + y + "'></td>" +
+            "<td data-x='5' data-y='" + y + "'></td>" +
+            "<td data-x='6' data-y='" + y + "'></td>" +
+            "<td data-x='7' data-y='" + y + "'></td>" +
+          "</tr>");
+    }
+
+    // draw black pawns
+    for (var i = 0; i <= 7; i++) {
```

Comment:
Oops forgot to drop a link. This should put you on the right track http://stackoverflow.com/questions/18162197/rails-pass-data-to-javascript